### PR TITLE
DEV-661 Fixed styling issues and made plus and minus button match lig…

### PR DESCRIPTION
**Pull request by willau24**
Fixed styling issues and made plus and minus button match light/dark mode

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
```
Comment (yuushazen) [inline .vscode/launch.json:10]: Should this be commited? The app doesn’t run on this port either

File: apps/web/src/components/PositionCard/index.tsx [hunk 2]
```diff
@@ -394,7 +394,7 @@ function FullPositionCard({
                 to={`/remove/${currencyId(currency0)}/${currencyId(currency1)}${isStableLP ? '?stable=1' : ''}`}
                 variant="primary"
                 width="100%"
-                mb="8px">
+                mb="0px">
```
Comment (yuushazen) [inline apps/web/src/components/PositionCard/index.tsx:397]: Can you just remove the mb property?

**Review by yuushazen — CHANGES_REQUESTED** [review]
Please fix lint errors

**Comment by yuushazen** [comment]
Please squash the commits before merging

**Comment by willau24** [comment]
> 

Ok I think I just squashed the other commits, should be good to go

**Review by yuushazen — APPROVED** [review]

**Review by vertotrade — APPROVED** [review]
LGTM

**Pull request merged**